index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Registration Form</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <header>
+      <h1 id="title">Microverse Registration Form</h1>
+      <p id="description">Please register your information</p>
+    </header>
+      <form id="suervey-form">          
+   <div>
```

Comment:
- [x] You have done a very good job requesting this change, however, Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

for example, you could say: :arrow_down: 

- [ ] Good job so far, :clap: , however, Please ensure that you use fieldset instead of `<div>` While `<div>` elements are versatile and can be used for general grouping or layout purposes, using a <fieldset> element when working with forms provides additional benefits in terms of accessibility, visual grouping, semantics, and form submission. It is recommended to use `<fieldset>` and `<legend>` elements when you have a logical grouping of form controls.